packages/client/README.md

```diff
@@ -114,257 +205,280 @@ const msg = {
-| autoConnect              | true                             | If set to `true`, the client connects automatically on the first call to `subscribe()`. Otherwise an explicit call to `connect()` is required.                                                                                                                                                                                          |
-| autoDisconnect           | true                             | If set to `true`, the client automatically disconnects when the last stream is unsubscribed. Otherwise the connection is left open and can be disconnected explicitly by calling `disconnect()`.                                                                                                                                        |
-| orderMessages            | true                             | If set to `true`, the subscriber handles messages in the correct order, requests missing messages and drops duplicates. Otherwise, the subscriber processes messages as they arrive without any check.                                                                                                                                  |
-
-## Authentication options
-
-Note: **Authenticating with an API key has been deprecated. Cryptographic keys/wallets is the only supported authentication method.**
+### Resend functionality with subscriptions
+By default `subscribe` will not resend historical data, only subscribe to real time messages. In order to fetch historical messages the stream needs to have [storage enabled](#storage).
 
-If you don't have an Ethereum account you can use the utility function `StreamrClient.generateEthereumAccount()`, which returns the address and private key of a fresh Ethereum account.
+Note that only one of the resend options can be used for a particular subscription. The default functionality is to resend nothing, only subscribe to messages from the subscription moment onwards.
 
-Authenticating with Ethereum also automatically creates an associated Streamr user, even if it doesn't already exist. Under the hood, the client will cryptographically sign a challenge to authenticate you as a Streamr user:
+One can either fetch the historical sent messages with the `resend` method:
+```js
+// Fetches the last 10 messages stored for the stream
+const resend1 = await streamr.resend(
+    streamId,
+    {
+        last: 10,
+    }, 
+    messageCallback
+)
+```
 
+Or fetch them and subscribe to new messages in the same call via a `subscribe` call:
 ```js
-const client = new StreamrClient({
-    auth: {
-        privateKey: 'your-private-key'
+// Fetches the last 10 messages and subscribes to the stream
+const sub1 = await streamr.subscribe({
```

Comment:
Should use `resend` in all examples? If we've separately mentioned that there are the two styles: `resend` and `subscribe`?